src/users/models/pydantic.py

```diff
@@ -19,3 +19,23 @@ class UserModel(BaseModel):
 
 class UserWithPassword(UserModel):
     hashed_password: str
+
+
+
+class UserAddressModel(BaseModel):
+    model_config = ConfigDict(from_attributes=True)
+
+    id: Union[int, None] = None
+    title: str
+
+
+class UserAddressModelDetail(UserAddressModel):
+    id: Union[int, None] = None
```

Comment:
```suggestion
    id: Optional[int] = None
```
Optional means that in case if value doesn't exist it will be None